robotcontainer.py

```diff
@@ -110,15 +110,17 @@ def __init__(self) -> None:
         self.driveToTarget = DriveToTarget(self.drive, constants.kAutoTargetOffset)
 
         # A routine that follows a set trajectory
-        self.fiveBRStandard = FiveBRStandard(self.drive, self.intake, self.indexer)
-        self.twoBLHangerbounce = TwoBLHangerbounce(
-            self.drive, self.intake, self.indexer
+        self.fiveBRStandard = FiveBRStandard(
+            self.shooter, self.drive, self.intake, self.indexer
         )
         self.twoBLHangerOuttake = TwoBLHangerOuttake(
-            self.drive, self.intake, self.indexer
+            self.shooter, self.drive, self.intake, self.indexer
         )
         self.fourBLNoninvasive = FourBLNoninvasive(
-            self.drive, self.intake, self.indexer
+            self.shooter, self.drive, self.intake, self.indexer
+        )
+        self.threeBRStandard = ThreeBRStandard(
+            self.shooter, self.drive, self.intake, self.indexer
```

Comment:
Reminder: we should change the default to whatever we decide to use most/is most consistent